### PR TITLE
UnzipFile - error

### PR DESCRIPTION
**Pull request by moishce**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-19760)

## Description
When there is a / at the beginning of the archived file
It removes it and sends a warning that it's removed
And we was returned it as an error

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by moishce** [comment]
[link to the failed test before the fix](https://app.circleci.com/pipelines/github/demisto/content/224329/workflows/a8c9ed40-1f89-457b-96e3-0250f0728bf8/jobs/631099)

File: Packs/CommonScripts/ReleaseNotes/1_10_47.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Scripts
+##### UnzipFile
+- Fixes an issue where warning were being returned as error.e *FiltersAndTransformers* pack.
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/ReleaseNotes/1_10_47.md:3]:
```suggestion
Fixed an issue where warnings were returned as errors when running this script on certain files.
```

File: Packs/CommonScripts/Scripts/UnzipFile/UnzipFile.py [hunk 2]
```diff
@@ -147,7 +147,8 @@ def extract_using_tarfile(file_path: str, dir_path: str, file_name: str) -> str:
     stdout, stderr = process.communicate()
     stdout = str(stdout)
     if stderr:
-        return_error(str(stderr))
+        if str(stderr, 'UTF-8') != "tar: removing leading '/' from member names\n":
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/Scripts/UnzipFile/UnzipFile.py:150]: I think we said we want to add a comment explaining this "weird" if statement

File: Packs/CommonScripts/Scripts/UnzipFile/UnzipFile_test.py [hunk 3]
```diff
@@ -237,3 +237,26 @@ def test_get_password_invalid():
         get_password(ARGS_BOTH_PASSWORDS_NOT_IDENTICAL)
         if not e:
             assert False
+
+
+def test_archive_with_slash_in_path():
+    """
+    Given
+    - valid tar.gz file slash in path
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/Scripts/UnzipFile/UnzipFile_test.py:245]:
```suggestion
    - valid tar.gz file with slash in path
```

File: Packs/CommonScripts/Scripts/UnzipFile/UnzipFile_test.py [hunk 4]
```diff
@@ -237,3 +237,26 @@ def test_get_password_invalid():
...
+    When
+    - run extract on the tar file and export the internal files to _dir
+    Then
+    - ensure no error was returned
+    """
+    zipped_file_object = {
+        'name': 'Archive_with_slash_in_path.tar.gz',
+        'path': 'data_test/Archive_with_slash_in_path.tar.gz'
+    }
+    # - empty folder _di
```
Comment (ShahafBenYakir) [inline Packs/CommonScripts/Scripts/UnzipFile/UnzipFile_test.py:256]:
```suggestion
    # - empty folder _dir
```

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**